bower.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "react-filterbar",
-  "version": "0.0.0",
```

Comment:
This is slightly outside the boundaries of my knowledge, but I agree with @jeremygrant - a version string seems necessary here. I know nothing about how bower uses version strings, but they are documented here http://bower.io/docs/creating-packages/, and it seems like, e.g. they can be used to specify dependencies. (And in general, giving a package a version number seems better than giving it no version number.)